build/configs/rtl8721csm/security_hal_test/defconfig

```diff
@@ -21,6 +21,7 @@ CONFIG_APPS_DIR="../apps"
 CONFIG_FRAMEWORK_DIR="../framework"
```

Comment:
Please undo change in file permissions (100644 -> 100755)

Reply:
removed this change